### PR TITLE
Fbx apply mesh transform

### PR DESCRIPTION
**Pull request by stigrus**

### CODE REVIEW

File: CadRevealComposer/CadRevealNode.cs [hunk 1]
```diff
@@ -48,6 +48,11 @@ public Box ToBoxPrimitive(uint treeIndex, Color color)
         var matrix = Matrix4x4.CreateScale(Extents) * Matrix4x4.CreateTranslation(Center);
         return new Box(matrix, treeIndex, color, this);
     }
+
+    public bool Same(BoundingBox other)
```
Comment (Strepto) [inline CadRevealComposer/CadRevealNode.cs:52]: Should: Add xml docs and test for this

File: CadRevealFbxProvider.Tests/FbxProviderTests.cs [hunk 2]
```diff
@@ -7,33 +7,33 @@ namespace CadRevealFbxProvider.Tests;
...
-using CadRevealFbxProvider.Attributes;
+using Attributes;
 using System.Numerics;
 
 [TestFixture]
 public class FbxProviderTests
 {
-    private static readonly DirectoryInfo outputDirectoryCorrect = new DirectoryInfo(@".\TestSamples\correct");
-    private static readonly DirectoryInfo inputDirectoryCorrect = new DirectoryInfo(@".\TestSamples\correct");
+    private static readonly DirectoryInfo OutputDirectoryCorrect = new("TestSamples//correct");
```
Comment (Strepto) [inline CadRevealFbxProvider.Tests/FbxProviderTests.cs:16]: Do we need double slashes here? Is there a diff between singel and double forward slash?

File: CadRevealFbxProvider.Tests/FbxNodeToCadRevealNodeConverterTests.cs [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+using CadRevealComposer.Primitives;
+using System.Numerics;
+
+[TestFixture]
+public class FbxNodeToCadRevealNodeConverterTests
+{
+    private static readonly string TestFile = new("TestSamples\\cube_and_instanced_cube_with_parent.fbx");
+
+    [Test]
+    public void CubeAndInstancedCubeParentedToBaseMeshAllWithTransforms_ConvertRecursive_VerifyCorrectTransformations()
```
Comment (Strepto) [inline CadRevealFbxProvider.Tests/FbxNodeToCadRevealNodeConverterTests.cs:16]: Consider adding a few comments on the expectations with inline comments in this test. I found it a bit hard to understand while reading.

File: CadRevealFbxProvider/FbxNodeToCadRevealNodeConverter.cs [hunk 4]
```diff
@@ -131,7 +128,9 @@ IReadOnlySet<IntPtr> geometriesThatShouldBeInstanced
     )
     {
         var nodeGeometryPtr = FbxMeshWrapper.GetMeshGeometryPtr(node);
-        var transform = FbxNodeWrapper.GetTransform(node);
+        var worldTransform = node.WorldTransform;
+
+        var translation = node.GetLocalTransform().Translation;
```
Comment (Strepto) [inline CadRevealFbxProvider/FbxNodeToCadRevealNodeConverter.cs:133]: Is translation used?

**Review by Strepto — APPROVED** [review]
Looks good to me except for a few suggestions above

File: CadRevealFbxProvider/FbxNodeToCadRevealNodeConverter.cs (hunk 4 above)
Comment (stigrus) [reply CadRevealFbxProvider/FbxNodeToCadRevealNodeConverter.cs:133]: Woops. Forgot to remove it. Used it for some debugging

**Pull request merged**